layouts/partials/footer.html

```diff
@@ -17,7 +17,7 @@
         {{ end }}
       </div>
       <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
-        {{ with .Site.Params.copyright }}<small class="text-white">Copyright &copy; {{ 2018 }} {{ .}} <br>{{ "Documentation Distributed under CC BY 4.0" }}</small>{{ end }}
+        {{ with .Site.Params.copyright }}<small class="text-white">Copyright &copy; {{ 2019 }} {{ .}} <br>{{ "Documentation Distributed under CC BY 4.0" }}</small>{{ end }}
```

Comment:
This is an interesting one. The copyright date should actually *not* be automatically set to the current year for pre-existing content. Because, if we set the copyright date to this year, it means we can no longer claim copyright for last year. If someone published our content last year, they could now claim we didn't claim copyright for it.

That said, there's a problem with having a single copyright statement for the site as a whole, because we're still publishing new content on the site this year, so the copyright date needs to be this year for that content.

I think we should claim copyright from 2018 (fixed date) to current year (auto-updated date). Something like this (code not tested):

```
Copyright &copy; {{ 2018 }} - {{ now.Year}} {{ .}} <br>
```

Reply:
Hello @sarahmaddox 
You are right, updated PR
https://github.com/gohugoio/hugoDocs/blob/master/themes/gohugoioTheme/layouts/partials/site-footer.html#L38